### PR TITLE
renamme fonction to function

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

**Comment by ngilain** [comment]
:+1:

File: WorkflowFunctionBundle/OpenOrchestraWorkflowFunctionBundle.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * Class OpenOrchestraWorkflowFunctionBundle
+ */
+class OpenOrchestraWorkflowFunctionBundle extends Bundle
+{
+    /**
+     * @param ContainerBuilder $container
+     */
+    public function build(ContainerBuilder $container)
```
Comment (nicolasThal) [inline WorkflowFunctionBundle/OpenOrchestraWorkflowFunctionBundle.php]: As there is no logic in this method, you don't have to override it

File: WorkflowFunctionBundle/Repository/WorkflowFunctionRepository.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ */
+class WorkflowFunctionRepository extends DocumentRepository implements WorkflowFunctionRepositoryInterface
+{
+    /**
+     * @return Collection
+     */
+    public function findAllWorkflowFunction()
+    {
+        $qb = $this->createQueryBuilder('w');
+        return $qb->getQuery()->execute();
```
Comment (nicolasThal) [inline WorkflowFunctionBundle/Repository/WorkflowFunctionRepository.php:22]: empty line before return.

You can just use the findAll method if you want here

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            "homepage": "https://github.com/open-orchestra/open-orchestra-function-bundle/graphs/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "open-orchestra/open-orchestra-model-interface": "*",
+        "doctrine/data-fixtures":"1.1.1",
+        "doctrine/mongodb-odm": "1.0.0-BETA12",
+        "doctrine/mongodb-odm-bundle" : "3.0.0-BETA6",
+        "stof/doctrine-extensions-bundle": "1.1.0"
```
Comment (nicolasThal) [inline composer.json:17]: do you really need all the stof/doctrine-extensions and not only the gedmo one ?

Comment (veolys-nanne) [inline composer.json:17]: How do you do that?, @nicolasThal

Comment (nicolasThal) [inline composer.json:17]: You can check if any logic given by the bundle is present in your code

Comment (veolys-nanne) [inline composer.json:17]: I'm not sure to understand. And what is Gedmo? @nicolasThal

Comment (nicolasThal) [inline composer.json:17]: all right for me

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "open-orchestra/open-orchestra-function-bundle",
```
Comment (nicolasThal) [inline composer.json]: change the name to open-orchestra/open-orchestra-workflow-function-bundle

**Pull request merged**